### DISCUSSION TITLE
Which Theme, color palette/schema is used in Waku doc?

### DISCUSSION THREAD

**Discussion by 2u841r** [wakujs/waku#747] [Q&A] [answered]
I would like to use it in my VS Code. 
![wakutheme](https://github.com/dai-shi/waku/assets/77893877/f71d4a86-de2e-4b17-8df1-ecf06dfcadd8)

**Comment by dai-shi** [comment]
@sandren ?

**Comment by sandren** [comment, marked as answer]
It's my favorite theme, [Lucy](https://marketplace.visualstudio.com/items?itemName=Hermitter.oh-lucy-vscode). 🙂
